src/eth/rpc.ts

```diff
@@ -30,6 +36,20 @@ const getDelegatorSnapshot = async (
     delegate,
   };
 };
+const isEOA = async (address: string) => {
+    return provider.getCode(address) === "0x"
+}
+
+const isOrchestrator = async (item) => {
+    return item.id === item.delegate.id;
+}
+
+const filterAddresses = async (arr) => {
+    const results = await Promise.all(
+        arr.map(item => (isEOA(item.id) || !isOrchestrator(item)))
```

Comment:
This conditional should use && since an account is snapshot eligible if it is an EOA *and* it is not an orchestrator.